### PR TITLE
Fix controller action not returning Promises

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do ?

Fix controller actions return, now all function return a Promise

Fix https://github.com/kuzzleio/kuzzle-plugin-probe-listener/issues/4

Related PR: https://github.com/kuzzleio/kuzzle-plugin-probe-listener/pull/5

## Other changes

  - Add a script to install plugins with `npm install` when Kuzzle start
  - add issue and pull request template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+-->
+
+<!--
+  IMPORTANT
+  Don't forget to add the corresponding "changelog:xxx" label to your PR.
+  This is part of our release process in order to generate the change log.
+-->
+
+
+## What does this PR do ?
```
Comment (scottinet) [inline .github/pull_request_template.md]:
```suggestion
## What does this PR do?
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+services:
+  - docker
+
+on:
+  branch: master
+
+env:
+  global:
+  - DOCKER_COMPOSE_VERSION=1.21.0
```
Comment (benoitvidis) [inline .travis.yml]: Do you have any evidence this environement variable is used by travis? As far as I can see, it is not and should not be needed anyway.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+group: edge
+
+services:
+  - docker
```
Comment (benoitvidis) [inline .travis.yml]: As far as I can see, docker is not needed.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+on:
+  branch: master
+
+env:
+  global:
+  - DOCKER_COMPOSE_VERSION=1.21.0
+
+before_install:
+  - sudo sysctl -w vm.max_map_count=262144
+  - sudo sysctl -w fs.inotify.max_user_watches=524288
```
Comment (benoitvidis) [inline .travis.yml]: We don't run either elasticsearch or any watcher. What are these kernel tweaks for?

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+group: edge
+
+services:
+  - docker
+
+on:
+  branch: master
```
Comment (benoitvidis) [inline .travis.yml]: hum... I don't think this is taken into account on a global scope by travis, which is a good thing anyway IMO as we want the tests to be run on other branches than master, no?

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+group: edge
+
```
Comment (benoitvidis) [inline .travis.yml]: Shouldn't we set the language to node instead of using the default ruby image?

File: docker-compose.yml [hunk 7]
```diff
@@ -2,12 +2,15 @@ version: '3'
 
 services:
   kuzzle:
+    command: sh -c '/var/app/run.sh'
```
Comment (benoitvidis) [inline docker-compose.yml]: nitpicking:
```suggestion
    command: /var/app/run.sh
``

File: docker-compose.yml [hunk 8]
```diff
@@ -27,6 +30,7 @@ services:
       - cluster.name=kuzzle
 
   kdc-kuzzle:
+    command: sh -c '/var/app/run.sh'
```
Comment (benoitvidis) [inline docker-compose.yml]:
```suggestion
    command: /var/app/run.sh
```

**Pull request merged**